tests/view_tests/tests/test_defaults.py

```diff
@@ -123,7 +123,7 @@ def test_bad_request(self):
     )
     def test_custom_bad_request_template(self):
         response = self.client.get("/raises400/")
-        self.assertIs(response.wsgi_request, response.context[-1].request)
+        self.assertIs(response.wsgi_request, response.context.request)
```

Comment:
This change is required because the test no longer renders two templates (email handler + error page), just the one (error page), and the test client flattens the `ContextList` into a single context:

https://github.com/django/django/blame/ca5cd3e3e8e53f15e68ccd727ec8fe719cc48099/django/test/client.py#L1108-L1109

I guess the same could happen to users’ test suites. Do you think I should add a release note? We typically don’t for optimizations, but...

Reply:
As this case is covered by the default Django settings (i.e. `ADMINS` is an empty list and `AdminEmailHandler` is in the default logging), I think it might be worth adding a release note.
I can be persuaded out of it (I see @felixxm approved earlier - maybe you have thoughts?).

Reply:
It's so tiny that I don't think a release note is needed.

Reply:
Alright thank you, let's not worry 👍